lib/modules/manager/gleam/range.ts

```diff
@@ -16,16 +16,6 @@ export function getRangeStrategy(config: RangeConfig): RangeStrategy {
     );
     return 'widen';
   }
-  if (rangeStrategy === 'update-lockfile') {
-    logger.warn(
-      'Unsupported rangeStrategy update-lockfile, defaulting to widen',
-    );
-    return 'widen';
-  }
-  if (rangeStrategy === 'in-range-only') {
-    logger.warn('Unsupported rangeStrategy in-range-only, defaulting to widen');
-    return 'widen';
-  }
   if (rangeStrategy !== 'auto') {
```

Comment:
why remove these lines?

Reply:
The purpose of this PR was to remove these restrictions for the Gleam Manager. Perhaps I overlooked something?

The new `gleam update <list of deps>` command allows us to update only the deps we need in the lockfile, respecting the range defined in the package file. I thought that covers both of these `rangeStrategy`'s.

Reply:
Noted